src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldFilterTypes/SelectionFieldFilterType.js

```diff
@@ -1,36 +1,35 @@
 // @flow
 import React from 'react';
 import type {ElementRef} from 'react';
-import {autorun, computed, toJS, untracked, when} from 'mobx';
+import {action, autorun, computed, observable, toJS, untracked, when} from 'mobx';
 import equals from 'fast-deep-equal';
 import MultiSelectionStore from '../../../stores/MultiSelectionStore';
 import MultiAutoComplete from '../../MultiAutoComplete';
+import ResourceMultiSelect from '../../ResourceMultiSelect';
 import AbstractFieldFilterType from './AbstractFieldFilterType';
 import selectionFieldFilterTypeStyles from './selectionFieldFilterType.scss';
 
+const TYPE_AUTO_COMPLETE = 'auto_complete';
+const TYPE_SELECT = 'select';
+
 class SelectionFieldFilterType extends AbstractFieldFilterType<?Array<string | number>> {
     selectionStore: MultiSelectionStore<string | number>;
     selectionStoreDisposer: () => void;
+    @observable selectValue: Array<string | number> = [];
```

Comment:
edit: took some time to understand why this is needed. but dont see a more intuitive solution at the moment. maybe add a comment that we need to buffer this to not send `SelectionStore` requests while the eselect is opened?